### DISCUSSION TITLE
Positron unable to start console in Remote SSH session

### DISCUSSION THREAD

**Discussion by azwadriqbal** [posit-dev/positron#5300] [Q&A] [answered]
Hello,

I am quite new to Positron and understand that the SSH functionality is relatively new, so I apologize if this issue was addressed elsewhere or will be addressed in the future.

I am attempting to connect remotely to our server via SSH and am able to successfully log in and see the files/folders/etc. However when attempting to interface with the console I see this:
```
Cannot read properties of undefined (reading 'createSession')
```
The console then hangs and no code can be executed regardless of interpreter, R version, etc.

Any help would be appreciated!

**Comment by jmcphers** [comment]
What version of Linux are you using on the remote server? Positron's interpreters currently don't work with Ubuntu <= 20, Debian <= 11, or RedHat <= 8. More details here: https://github.com/posit-dev/positron/discussions/5296

**Comment by azwadriqbal** [comment]
Hi Jonathan, our server is running Rocky 9, which I believe is based on RedHat source code if I'm not mistaken?

**Reply by jmcphers** [reply, marked as answer]
Yup, it is. Can you try enabling our experimental kernel supervisor to see if that fixes it for you? Look in the settings for 'Kallichore', turn it on, then quit and restart Positron. 

(This experimental supervisor is compatible with more Linux distributions than the previous one, and will become the default in Positron's 2024.12 release)

**Reply by azwadriqbal** [reply]
that appears to have worked, thank you!
